x-pack/legacy/plugins/ingest_manager/index.ts

```diff
@@ -8,6 +8,7 @@ import { resolve } from 'path';
 export function ingestManager(kibana: any) {
   return new kibana.Plugin({
     id: 'ingestManager',
+    require: ['kibana', 'elasticsearch', 'xpack_main'],
```

Comment:
@elastic/ingest-management
Could you please help me to deal with that broken CI test `apis Fleet Endpoints fleet_setup should create a fleet_enroll user and role` ?

Not sure that it's somehow related to my changes but looks like my PR forced that problem #68568

Reply:
on my local it works fine.
![image](https://user-images.githubusercontent.com/20072247/84914034-d365d900-b0c3-11ea-80b3-fa1d53802d22.png)

Reply:
@elastic/ingest-management Could someone help out here, please?

Reply:
For this error, we currently have an external dependency in our tests (to the package registry), we are working to have it running locally, and avoid these one time errors.

Reply:
@nchaulet  I changed the order of the tests a bit. Now everything passes. Could you take a look at this PR?

I was able to reproduce this problem locally, but it seems now everything is working correctly